examples/twr_aloha/control.c

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ *
+ * @}
+ */
+
+#include <assert.h>
+#include <math.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "uwb/uwb.h"
+#include "uwb_rng/uwb_rng.h"
+#include "dpl/dpl.h"
+#include "control.h"
+
+#include "shell_commands.h"
+#include "shell.h"
+#include "xtimer.h"
```

Comment:
xtimer is not used in this file so the include could be removed
```suggestion
```

Reply:
There is `uint32_t utime = xtimer_now_usec();`

Reply:
Ah yes, sorry!